tests/test_utils.py

```diff
@@ -194,3 +195,91 @@ def test_get_prefixes(self):
+
+    def test_standardize_metadata_upgrade_multivalued_single(self):
+        """Test standardizing upgrades a string to a list."""
+        metadata = {"creator_id": "orcid:0000-0003-4423-4370"}
+        msdf = MappingSetDataFrame(df=pd.DataFrame(), converter=Converter([]), metadata=metadata)
+        msdf._standardize_metadata_references()
+        self.assertEqual({"creator_id": ["orcid:0000-0003-4423-4370"]}, msdf.metadata)
+
+    def test_standardize_metadata_upgrade_multivalued_multiple(self):
+        """Test standardizing upgrades a string to a list."""
+        metadata = {"creator_id": "orcid:0000-0003-4423-4370 | orcid:0000-0002-6601-2165"}
+        msdf = MappingSetDataFrame(df=pd.DataFrame(), converter=Converter([]), metadata=metadata)
+        msdf._standardize_metadata_references()
+        self.assertEqual(
+            {"creator_id": ["orcid:0000-0003-4423-4370", "orcid:0000-0002-6601-2165"]},
+            msdf.metadata,
+        )
+
+    def test_standardize_metadata_multivalued(self):
+        """Test standardizing upgrades a string to a list."""
+        metadata = {"creator_id": ["orcid:0000-0003-4423-4370", "orcid:0000-0002-6601-2165"]}
+        msdf = MappingSetDataFrame(df=pd.DataFrame(), converter=Converter([]), metadata=metadata)
+        msdf._standardize_metadata_references()
+        self.assertEqual(
+            {"creator_id": ["orcid:0000-0003-4423-4370", "orcid:0000-0002-6601-2165"]},
+            msdf.metadata,
+        )
+
+    def test_standardize_metadata_multivalued_type_error(self):
+        """Test raising on a  non-string, non-list object given to a multivalued slot."""
+        metadata = {"creator_id": object()}
+        msdf = MappingSetDataFrame(df=pd.DataFrame(), converter=Converter([]), metadata=metadata)
+        with self.assertRaises(TypeError):
+            msdf._standardize_metadata_references()
+
+    def test_standardize_error_non_list(self):
+        """Test that a type error is raised when metadata is presented that should not be a list."""
+        metadata = {"mapping_registry_id": ["https://example.org/r1", "https://example.org/r2"]}
+        msdf = MappingSetDataFrame(df=pd.DataFrame(), converter=Converter([]), metadata=metadata)
+        with self.assertRaises(TypeError):
+            msdf._standardize_metadata_references()
+
+    def test_standardize_coerce_list(self):
+        """Test that a metadata field that should not be a list gets coerced to a single element."""
+        metadata = {"mapping_registry_id": ["https://example.org/r1"]}
+        msdf = MappingSetDataFrame(df=pd.DataFrame(), converter=Converter([]), metadata=metadata)
+        msdf._standardize_metadata_references()
+        self.assertEqual({"mapping_registry_id": "https://example.org/r1"}, msdf.metadata)
+
+    def test_standardize_metdata_delete_empty(self):
+        """Test that an element that should not be a list but is empty just gets deleted."""
+        metadata = {"mapping_registry_id": []}
```

Comment:
`mapping_registry_id` is not a mapping set element. Use `mapping_set_id` instead.

Reply:
```suggestion
        metadata = {"mapping_set_id": []}
```

Reply:
(you do this multiple times so I let you fix this on your branch)

Reply:
oof, changing this breaks all of the tests since `mapping_set_id` has different properties. Is this necessary?

Reply:
Well, it is not at all in the schema -  don't you want at least to test an element that is in the schema?

https://mapping-commons.github.io/sssom/MappingSet/

Reply:
oh, I see.

Reply:
@matentzn great feedback, thank you. I updated the implementation to take this into account in e16db81 (also with tests)